source/developer/developer-setup.md

```diff
@@ -4,44 +4,46 @@ Developer Machine Setup
 ### Mac OS X ###
 
 1. Download and set up Docker Toolbox
-	1. Follow the instructions at http://docs.docker.com/installation/mac/
```

Comment:
You shouldn't need the backquotes around the URL here. Just having `[http://docs.docker.com/installation/mac/](http://docs.docker.com/installation/mac/)` should be fine. Same with the other URLs